packages/runtime/src/client/options.ts

```
@@ -62,11 +65,17 @@ export type ClientOptions<Schema extends SchemaDef> = {
     log?: KyselyConfig['log'];
 } & (HasComputedFields<Schema> extends true
     ? {
+          /**
+           * Computed field definitions.
+           */
           computedFields: ComputedFieldsOptions<Schema>;
       }
     : {}) &
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)